lang/system/pl.xml

```diff
@@ -308,44 +556,268 @@ System Vita3K jest gotowy!</completed_setup>
 				<mm_dd_yyyy>MM/DD/RRRR</mm_dd_yyyy>
 			</date_format>
 			<time_format name="Format godziny">
-				<clock_12_hour>Zegar 12-godz</clock_12_hour>
-				<clock_24_hour>Zegar 24-godz</clock_24_hour>
+				<clock_12_hour>Zegar 12-godzinowy</clock_12_hour>
+				<clock_24_hour>Zegar 24-godzinowy</clock_24_hour>
```

Comment:
Don't touch it, revert change for already translate already present

Reply:
@Zangetsu38 if you want to use short form, use the damn dot.
Polish langauge has rule, if shortcut does not end with last letter of this word, needs a dot at the end.

Reply:
Is official translation, juste revert